scipy/spatial/_spherical_voronoi.py

```diff
@@ -253,15 +253,15 @@ def _calc_vertices_regions(self):
         )
 
         # calculate regions from triangulation
-        generator_indices = np.arange(self.points.shape[0])
-        filter_tuple = np.where((np.expand_dims(self._tri.simplices,
-                                -1) == generator_indices).any(axis=1))
+        simplex_indices = np.arange(self._tri.simplices.shape[0])
+        tri_indices = np.column_stack([simplex_indices, simplex_indices,
+            simplex_indices]).ravel()
+        point_indices = self._tri.simplices.ravel()
 
-        list_tuples_associations = zip(filter_tuple[1],
-                                       filter_tuple[0])
+        array_associations = np.dstack((point_indices, tri_indices))[0]
 
-        list_tuples_associations = sorted(list_tuples_associations,
-                                          key=lambda t: t[0])
+        list_tuples_associations = \
```

Comment:
backslashes are frowned upon (ugly, and too easy to get syntax errors by just adding a space after it). in this case, better to break after ``argsort(``